src/generators/minecraft-mutant-character/MutantCharacter.res

```diff
@@ -0,0 +1,507 @@
+let requireImage = fileName => Generator.requireImage("./images/" ++ fileName)
+let requireTexture = fileName => Generator.requireImage("./textures/" ++ fileName)
+
+let id = "mutantgenerator"
+
+let name = "Mutant Character"
+let video: Generator.videoDef = {url: "https://www.youtube.com/embed/DVzumgRinjY"}
+let thumbnail: Generator.thumnbnailDef = {
+  url: Generator.requireImage("./thumbnail/thumbnail.jpeg"),
+}
+let images: array<Generator.imageDef> = [
+  {id: "Page1", url: requireImage("Page1.png")},
+  {id: "Page2", url: requireImage("Page2.png")},
+  {id: "Page3", url: requireImage("Page3.png")},
+  {id: "Page4", url: requireImage("Page4.png")},
+  {id: "Hole1", url: requireImage("Hole1.png")},
+  {id: "hole2", url: requireImage("hole2.png")},
+  {id: "hole3", url: requireImage("hole3.png")},
```

Comment:
For consistency, would be good to rename `hole2` and `hole3` to `Hole2` and `Hole3`

Reply:
Done